src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -64,6 +71,16 @@
 import java.util.List;
 
 public class PlatformInterop {
+    private static final int MAUI_GLIDE_CLEARED_TAG_ID = R.id.maui_glide_cleared_tag;
+    
+    public static void setGlideClearedTag(View view, boolean isCleared) {
+        view.setTag(MAUI_GLIDE_CLEARED_TAG_ID, isCleared);
+    }
+    
+    public static boolean getGlideClearedTag(View view) {
+        Object tag = view.getTag(MAUI_GLIDE_CLEARED_TAG_ID);
+        return tag != null && (boolean)tag;
```

Comment:
Does this need to check `instanceof` in case it is another type?
```suggestion
        return tag != null && tag instanceof boolean && (boolean)tag;
```